.storybook/main.ts

```diff
@@ -21,6 +21,7 @@ const config: { webpackFinal: (config: any) => Promise<any>; stories: string[];
     framework: {
         name: '@storybook/react-webpack5',
     },
+    staticDirs: ['../public'],
```

Comment:
need to add this so MSW could be loaded to the same static directory as storybook to mock net work request